dell/xps/15-9570/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, ... }:
+
+{
+  imports = [
+    ../../../common/cpu/intel
+    ../../../common/pc/laptop
+  ];
+
+  # TODO: boot loader
+  boot.loader.systemd-boot.enable = lib.mkDefault true;
+  boot.loader.efi.canTouchEfiVariables = lib.mkDefault true;
```

Comment:
Do not enable this by default. It should be only used during installation.
Enabling it by default will wear out the NVRAM since it would write new entries on every `nixos-rebuild switch`.